lab-anthony/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lab-anthony",
+  "version": "0.1.0",
+  "description": "using gulp to run mocha/assert/eslint",
+  "main": "greet-cli.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "@ascasson",
+  "license": "ISC",
```

Comment:
Looks good :), always remember to make sure you're using the license you want to use.